src/components/Article.jsx

```diff
@@ -1 +1,31 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, date, text }) => {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString(
+    'en-US',
+    {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+    },
+  );
```

Comment:
maybe you can evet extract this logic in a method `processDate`
which can be called like this
`
const {
  dateISO,
  dateLocaleString,
} = processDate(date);
`